definitions/npm/angular_v1.5.x/flow_v0.47.x-/test_angular_v1.5.x.js

```diff
@@ -108,7 +108,7 @@ describe("service", () => {
       "bar",
       "bazz",
       (bar, bazz) => {
-        return a => a + 1;
+        return { foo: "bar" };
```

Comment:
One side note: this was a test defect: `Object` was allowing a number to be returned so this never threw an error until I changed `Object` to `{}`.